bitnami/redis-cluster/7.0/debian-11/prebuildfs/opt/bitnami/scripts/libfile.sh

```diff
@@ -34,7 +34,13 @@ replace_in_file() {
     if [[ $posix_regex = true ]]; then
         result="$(sed -E "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
     else
-        result="$(sed "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
+     #Added substitute_regex matching to the environment, matching the save keyword to write the substitute_regex value into the redis.conf file
```

Comment:
This script is widely used and shared with almost all Bitnami containers, so any specific need should be refactorized into internal functions on each asset-specific library (in this case [libredis.sh](https://github.com/bitnami/containers/pull/23936/files#diff-2008d04100dc9499a52993f1d95e6819d6ded60a8934256b79398e191222c08e)).